src/VehicleSetup/JoystickConfig.qml

```diff
@@ -453,6 +453,14 @@ SetupPage {
 
                                     onClicked: _activeJoystick.throttleMode = 1
                                 }
+
+                                QGCCheckBox {
```

Comment:
It would seem the answer is yes https://github.com/mavlink/qgroundcontrol/pull/5185/files#diff-7179de35ff534916a12fd2e757566fe8R464